notebooks/problem-sets/Quantum states and operations/01.Single_qubit_quantum_states.ipynb

```diff
@@ -0,0 +1,72 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d7613bc7",
+   "metadata": {},
+   "source": [
+    "## Problem 1\n",
+    "\n",
+    "<!-- ::: q-quiz(goal=\"test-problem\") -->\n",
```

Comment:
@amirhassank the `goal` should be a unique value. it appears you are using `test-problem` multiple times